otherlibs/dynlink/natdynlink.ml

```diff
@@ -16,11 +16,15 @@
 
 (* Dynamic loading of .cmx files *)
 
+open Cmx_format
+
 type handle
 
-external ndl_open: string -> bool -> handle * bytes = "caml_natdynlink_open"
+type globals_map = (string * Digest.t * Digest.t * string list) list
```

Comment:
Can we use a record type instead of this tuple?  It's quite unclear what the components are...